scripts/core/copy.yml

```diff
@@ -459,6 +459,12 @@ etors:
       desc: "hints that memory should be cached"
     - name: BIAS_UNCACHED
       desc: "hints that memory should be not be cached"
+    - name: SET_SYSTEM_MEMORY_PREFERRED_LOCATION
+      version: "1.7"
+      desc: "hint that the preferred memory location is the host"
```

Comment:
hint that the preferred memory location is host memory

Reply:
thanks. changing.